src/components/navi-button/types.ts

```diff
@@ -0,0 +1,3 @@
+import { ButtonProps } from '../button';
+
+export type NaviButtonProps = Omit<ButtonProps, 'variant' | 'fill' | 'name'>;
```

Comment:
you have to remove `'name'` here, so it gets available for the NaviButton, since you use it newly. Then also the pipelines will work